From 1b5d5043b5b9c4dcf347cba4ede40e8e5281e053 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Mon, 8 Jan 2024 13:30:03 +0000
Subject: [PATCH 1/1] package/linux: Provide launchable tags in appstream
 metainfo file

By default the .desktop file gets installed to sunshine.desktop, as
such provide the launchable tag for it to help `appstream-builder`
"find" the .desktop file.

Provide URL as well.

Spec: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-launchable
---
 packaging/linux/sunshine.appdata.xml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/packaging/linux/sunshine.appdata.xml b/packaging/linux/sunshine.appdata.xml
index fdb6b02..fb6e850 100644
--- a/packaging/linux/sunshine.appdata.xml
+++ b/packaging/linux/sunshine.appdata.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
-    <id>@PROJECT_NAME@.desktop</id>
+    <id>@PROJECT_NAME@</id>
     <metadata_license>@PROJECT_LICENSE@</metadata_license>
     <project_license>@PROJECT_LICENSE@</project_license>
     <name>@PROJECT_NAME@</name>
@@ -11,6 +11,8 @@
             @PROJECT_LONG_DESCRIPTION@
         </p>
     </description>
+    <launchable type="desktop-id">sunshine.desktop</launchable>
+    <launchable type="url">https://localhost:47990</launchable>
     <screenshots>
         <screenshot type="default">
             <image>https://app.lizardbyte.dev/Sunshine/assets/images/AdobeStock_305732536_1920x1280.jpg</image>
-- 
2.43.0

